Task70/YandexPages/HomePage.cs

```diff
@@ -58,5 +61,22 @@ public void Login(string username, string password)
             passwordPage.EnterPassword(password);
             passwordPage.ClickLoginButton();
         }
+
+        public void TakeScreenshot(string testCase)
+        {
+            try
```

Comment:
The calling of try/catch construction there is unnecessary because it will take screenshot anyway